server/lib/transferwise.ts

```diff
@@ -0,0 +1,124 @@
+import axios from 'axios';
+import config from 'config';
+import { get, omitBy, isNull } from 'lodash/fp';
+
+import { Quote, RecipientAccount } from '../types/transferwise';
+
+const compactRecipientDetails = omitBy(isNull);
+
+interface CreateQuote {
+  profileId: number;
+  sourceCurrency: string;
+  targetCurrency: string;
+  targetAmount?: number;
+  sourceAmount?: number;
+}
+export const createQuote = async (
+  token: string,
+  { profileId: profile, sourceCurrency, targetCurrency, targetAmount, sourceAmount }: CreateQuote,
+): Promise<Quote> =>
+  axios
+    .post(
+      `${config.transferwise.api}/v1/quotes`,
+      {
+        profile,
+        source: sourceCurrency,
+        target: targetCurrency,
+        rateType: 'FIXED',
+        type: 'BALANCE_PAYOUT',
+        targetAmount,
+        sourceAmount,
+      },
+      { headers: { Authorization: `Bearer ${token}` } },
+    )
+    .then(get('data'));
```

Comment:
Maybe add a catch to log the error and return something that we can display to the user (I can imagine that errors may contain private infos)